src/navigators/BottomNavigator.tsx

```diff
@@ -63,6 +68,9 @@ const BottomNavigator = () => {
         options={{
           title: getString('browse'),
           tabBarIcon: 'compass-outline',
+          tabBarBadge: pluginsWithUpdate
+            ? pluginsWithUpdate.toString()
+            : undefined,
```

Comment:
Show red badge informing user if plugins need update

Reply:
No pls, a red badge is annoying

Reply:
![image](https://github.com/user-attachments/assets/84f69ca4-dbe0-4ba7-a0f1-ad32ce4944c1)
it looks like this

Reply:
(emulator's screen is uber low res, text should look better on an actual phone)

Reply:
![image](https://github.com/user-attachments/assets/04989d5b-9ab7-4033-8a53-64382efd6d2c)
solves this issue

Reply:
Can you make it less red?

Reply:
![Screenshot_01_1](https://github.com/user-attachments/assets/1ebd5698-d95f-4057-ad00-16192f32d4a9)

Reply:
> is that not the point, to make users update their plugins so they dont complain abt the plugins not working

Yeah, but if they willingly go to settings to disable the badge thing it should be fine.

Reply:
> Can you make it less red? Like tachi

doesent seem to be possible for some reason, not sure why, they have style options in their documentation but it doesent work in app

Reply:
Hiding nav label feature works tho

Reply:
trt asking skilgg on ds